server/pom.xml

```diff
@@ -280,9 +280,7 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>${maven.surefire.plugin.version}</version>
                 <configuration>
-                    <forkMode>always</forkMode>
-                    <forkCount>3</forkCount>
-                    <reuseForks>true</reuseForks>
+                    <parallel>all</parallel>
```

Comment:
Any impact on Jenkins memory, during dev-branch build?

Reply:
I think that limit is 2G according this:
```<argLine>${argLine} -Xmx2048m -XX:MaxPermSize=1024m -Djava.awt.headless=true</argLine>```